src/vs/workbench/services/extensions/common/extensionPoints.ts

```diff
@@ -517,6 +517,7 @@ export class ExtensionScannerInput {
 		public readonly absoluteFolderPath: string,
 		public readonly isBuiltin: boolean,
 		public readonly isUnderDevelopment: boolean,
+		public readonly targetPlatform: TargetPlatform,
```

Comment:
`targetPlatform` should be checked for equality in the `ExtensionScannerInput.equals` method below.